### PR TITLE
[SPARK-30776][ML][FOLLOWUP] FValue clean up

### PR DESCRIPTION
**Pull request by zhengruifeng**
### What changes were proposed in this pull request?
remove unused variables;

### Why are the changes needed?
remove unused variables;

### Does this PR introduce any user-facing change?
No

### How was this patch tested?
existing testsuites

### CODE REVIEW

File: mllib/src/main/scala/org/apache/spark/ml/feature/FValueSelector.scala [hunk 1]
```diff
@@ -272,10 +269,13 @@ class FValueSelectorModel private[ml](
     val selectedFeatures: Array[Int])
   extends Model[FValueSelectorModel] with FValueSelectorParams with MLWritable {
 
-  var prev = -1
```
Comment (zhengruifeng) [inline mllib/src/main/scala/org/apache/spark/ml/feature/FValueSelector.scala]: `prev` was exposed:
image

File: mllib/src/test/scala/org/apache/spark/ml/feature/FValueSelectorSuite.scala [hunk 2]
```diff
@@ -127,21 +127,18 @@ class FValueSelectorSuite extends MLTest with DefaultReadWriteTest {
   test("Test FValue selector: numTopFeatures") {
     val selector = new FValueSelector()
       .setOutputCol("filtered").setSelectorType("numTopFeatures").setNumTopFeatures(1)
-    val model = selector.fit(dataset)
```
Comment (zhengruifeng) [inline mllib/src/test/scala/org/apache/spark/ml/feature/FValueSelectorSuite.scala]: `model` here is never used, and a model will be fitted in following `testSelector`

**Comment by zhengruifeng** [comment]
retest this please

**Comment by huaxingao** [comment]
LGTM

**Review by srowen — APPROVED** [review]

**Comment by zhengruifeng** [comment]
Merged to master, thanks all

**Pull request closed without merging**